tutorials/real-time-analytics-tutorial.rst

```diff
@@ -0,0 +1,153 @@
+    CREATE INDEX payload_index ON github_events USING GIN (payload jsonb_path_ops);
+
+Distributing tables and loading data
+------------------------------------
+
+We will now go ahead and tell Citus to distribute these tables across the nodes in the cluster. To do so,
+you can run :code:`create_distributed_table` and specify the table you want to shard and the column you want to shard on.
+In this case, we will shard all the tables on :code:`user_id`.                             
+                                                                                          
+::
+    
+    SELECT create_distributed_table('github_users', 'user_id');                                       
+    SELECT create_distributed_table('github_events', 'user_id');                               
+                                                                                          
+Sharding all tables on the user identifier allows Citus to :ref:`colocate <colocation>` these tables together,
+and allows for efficient joins and distributed roll-ups. You can learn more about the benefits of this approach `here <https://www.citusdata.com/blog/2016/11/29/event-aggregation-at-scale-with-postgresql/>`_.
+                                                                                          
+Then, you can go ahead and load the data we downloaded into the tables using the standard PostgreSQL :code:`\COPY` command.
+Please make sure that you specify the correct file path if you downloaded the file to a different location.
+
+::
+                                                                                          
+    \copy github_users from 'users.csv' with csv;                                                     
+    \copy github_events from 'events.csv' with csv;                                                     
+
+
+Running queries
+----------------
+
+Now that we have loaded data into the tables, let's go ahead and run some queries. First, let's check how many users we have in our distributed database.
+
+::
+                                                                                          
+    SELECT count(*) FROM github_users;
+    
+Now, let's analyze Github push events in our data. We will first find the number of distinct commits in push events and then compute the total number of commits by each minute.
+
+::
+                                                                                          
+    SELECT date_trunc('minute', created_at) AS minute,
+           sum((payload->>'distinct_size')::int) AS num_commits
+    FROM github_events
+    WHERE event_type = 'PushEvent'
+    GROUP BY minute
+    ORDER BY minute;                                                                                          
+
+We also have a users table. We can also easily join the users with events, and find users who created the most repositories. 
```

Comment:
(Minor changes -- feel free to ignore if you like the current one better)

We also have a users table. We can easily join the users with events, and find the top 10 users who created the most repositories.